gradle/osgi.gradle

```diff
@@ -16,7 +16,7 @@ afterEvaluate {
                     "org.mockito.*;version=${version}"
 
             instruction 'Import-Package',
-                    'net.bytebuddy.*;version=[1.4.26,2.0)',
+                    'net.bytebuddy.*;version="[1.4.26,2.0)"',
```

Comment:
@bric3 nobody want's to introduce an infinite boundary, i requested to use the short form